### DISCUSSION TITLE
Impact of Retrospective Marking as “Irrelevant” on Scoping Review Screening Process

### DISCUSSION THREAD

**Discussion by DianeMak** [asreview/asreview#1910] [Q&A] [answered]
I’m working on a scoping review. I have essentially screened my dataset (using oracle mode and default settings) up to our stopping rule. In retrospect, however, we want to mark some records (I think about 10% of the relevant records) that we have now labeled as "relevant" as "irrelevant" based on the **type** of study (not based on the subject). Does this retrospective marking of records as “irrelevant” affect the offering of new records if we were to continue screening now? 
The underlying question is: We are wondering if we would still find new relevant records by retrospectively applying this new exclusion criterion; or does it not work that way, and would it be better to for example restart the screening from scratch with the same dataset and this new exclusion criterion, in order to properly train the active learning model? I would love to hear your ideas on how to approach this.

**Comment by jteijema** [comment, marked as answer]
Hi @DianeMak!

You can definitely retrospectively apply the new exclusion criterion without starting over. However, I would recommend continuing screening after this moment, setting a new stopping rule.

In cases where users want to add an inclusion criteria, or remove an exclusion criteria, I'd recommend starting over; this is because some records might falsely be labeled irrelevant, and these noise labels will either stay in the dataset, or you'll be forced to check the pile of irrelevant labeled records (which is akin to starting over).

However, in the case of retrospectively excluding papers, you'll have to recheck the inclusions, which is feasible without starting over.

As for the introduced noise; the machine learning algorithm is trained from scratch every iteration. Adding a new exclusion criteria, in combination with double-checking the inclusions, means all noise is removed from the dataset in the next iteration. This might result in new papers being shown which were previously hidden. Therefore I'd recommend continuing the screening (for example until the stopping rule is activated again).

**Reply by jteijema** [reply]
tl;dr
You can apply a new exclusion criterion retrospectively and continue screening without starting over, as long as you check your inclusions and set a new stopping rule. Changing irrelevant to relevant is not recommended, but changing relevant to irrelevant is fine. Rechecking inclusions and retraining an iteration will remove noise, potentially surfacing new papers.

**Reply by jteijema** [reply]
One more explanation; Inclusion criteria pull records to the front. Exclusion criteria push papers to the back. Adding a new exclusion criteria won't pull any new papers out of the pile, but it will push some papers out of the way, moving the bulk of the papers forward. 

This is why I'd recommend screening some more; your old stopping rule is no longer valid, because in reality you screened less than the stopping rule implies. You screened some records that should've been pushed back, therefore you did not see some of the papers that the stopping rule would've included had you started with these criteria.

**Reply by DianeMak** [reply]
Thanks for your reply @jteijema! This makes it very clear, and we will proceed with the screening.

**Reply by Rensvandeschoot** [reply]
> the machine learning algorithm is trained from scratch every iteration.

see https://doi.org/10.3390/app14093842
